src/steps/11-GeneratePackageDocuments/UploadSignedDocuments.vue

```diff
@@ -172,6 +174,13 @@ export default class UploadSignedDocuments extends SaveOnLeave {
   @Watch("uploadedFiles")
   private async filesUploaded(): Promise<void>{
     await this.setDisableContinue();
+    console.log("Is current user a contributor?", AcquisitionPackage.currentUserIsContributor);
```

Comment:
Make sure to remove these

Reply:
console logs removed in most recent commit